Pages/SeamlessPostingLoginCallback.html.subtemplate

```diff
@@ -0,0 +1,7 @@
+{% extends "templates/base.html.template" %}
+
+{% block head %}
+  <script>
+    window.close();
```

Comment:
You can't put inline scripts in the chrome extension context.

When logging in from the seamless posting button, the login window redirects here and the window doesn't close.